site/class-1-load/faena/index.csv

```diff
@@ -0,0 +1,4 @@
+day, dinner
+"2017-09-24", Pizza
+"2017-09-23", Chicken Sateh
+"2017-09-22", Pasta Pesto with Chicken
```

Comment:
Ah, yes we’ll cover this in the next class, but you can’t have a space after `,`, so if you’d use this, it should work:

```
day,dinner
2017-09-24,Pizza
2017-09-23,Chicken Sateh
2017-09-22,Pasta Pesto with Chicken 
```